application/testing/CMakeLists.txt

```diff
@@ -815,7 +820,7 @@ f3d_test(NAME TestInteractionCycleAnimationNoAnimation DATA cow.vtp INTERACTION
 
 f3d_test(NAME TestInteractionDropFiles ARGS -n INTERACTION_CONFIGURE UI)#X;DropEvent cow.vtp;DropEvent dragon.vtu suzanne.stl;
 f3d_test(NAME TestInteractionMultiFileDrop ARGS --multi-file-mode=all -e INTERACTION_CONFIGURE) #DropEvent mb_1_0.vtp mb_2_0.vtp
-f3d_test(NAME TestInteractionDropSameFiles ARGS -x INTERACTION_CONFIGURE)#DropEvent cow.vtp;#DropEvent dragon.vtu;#DropEvent cow.vtp#DropEvent cow.vtp;
+f3d_test(NAME TestInteractionDropSameFiles ARGS -x INTERACTION_CONFIGURE UI) #DropEvent cow.vtp;#DropEvent dragon.vtu;#DropEvent cow.vtp#DropEvent cow.vtp;
```

Comment:
why UI ?

Reply:
because there's a warning badge

Reply:
Hum this is something I did not consider, this reduce our testing in the non-optional build. With that in consideration, should we `--verbose=error` ?

Reply:
https://github.com/f3d-app/f3d/pull/1748#discussion_r1869440066 ^^

Reply:
Yeah, I know, sorry about that.